src/ImageGallery.jsx

```diff
@@ -1161,6 +1161,7 @@ export default class ImageGallery extends React.Component {
                 <div
                   className='image-gallery-thumbnails'
                   ref={i => this._thumbnailsWrapper = i}
+                  data-tag_section="preview_carousel"
```

Comment:
should this be a default prop? then set to prop?

Reply:
I thought we preferred not to use props when possible. And this _should_ remain as "preview_carousel" indefinitely.